Keas.Mvc/ClientApp/components/Keys/KeyContainer.tsx

```diff
@@ -16,11 +16,11 @@ interface IState {
 }
 
 interface IProps {
-  keyAssigned?: (type: string, spaceId: number, personId: number, created: boolean, assigned: boolean) => void;
-  keyRevoked?: (type: string, spaceId: number, personId: number) => void;
+  keyInUseUpdated: (type: string, spaceId: number, personId: number, count: number) => void;
+  keyTotalUpdated: (type: string, spaceId: number, personId: number, count: number) => void;
```

Comment:
@laholstege why bother calling it `keyTotalUpdated` when (1) it's already in the key container, and (2) you are passing "key" as the first param anyway.  Same thing with other asset types